driver-core/src/test/java/com/datastax/driver/core/DataTypeTest.java

```diff
@@ -317,22 +317,4 @@ public void serializeDeserializeCollectionsTest(ProtocolVersion version) {
             fail("This should not have worked");
         } catch (InvalidTypeException e) { /* That's what we want */ }
     }
-
-    @Test(groups = "unit")
-    public void should_not_return_v4_types_in_all_primitive_types_with_v3() {
-        Set<DataType> dataTypes = DataType.allPrimitiveTypes(ProtocolVersion.V3);
-
-        // Ensure it does not contain specific newer types tinyint and smallint.
-        assertThat(dataTypes).doesNotContainAnyElementsOf(newArrayList(DataType.tinyint(), DataType.smallint()));
-
-        // Ensure all values are <= V3.
-        for (DataType dataType : dataTypes) {
-            assertThat(dataType.getName().minProtocolVersion).isLessThanOrEqualTo(ProtocolVersion.V3);
-        }
-    }
-
-    @Test(groups = "unit")
-    public void should_return_same_elements_with_all_primitive_types_using_latest_protocol_version() {
-        assertThat(DataType.allPrimitiveTypes(ProtocolVersion.NEWEST_SUPPORTED)).isEqualTo(DataType.allPrimitiveTypes());
-    }
```

Comment:
Removed these two tests, as they cover a method that has been moved to TestUtils.